tools/schema/cardTreeDirectorySchema.json

```diff
@@ -390,6 +390,7 @@
                   "description": "A logic program that only includes all the card-specific logic programs",
                   "type": "object"
                 },
+                "imports.lp": {},
```

Comment:
Why? I thought I removed imports.lp.

Reply:
Actually I think what this is about. Imports.lp is not generated anymore, but it is also not removed and that's why validation complains about it

Reply:
Yeah, and it shouldn't be part of this PR, but otherwise tests won't pass.